exporters/fluentd/cmake/opentelemetry-cpp.cmake

```diff
@@ -13,6 +13,8 @@ function(build_opentelemetry)
   set(opentelemetry_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/opentelemetry-cpp")
   set(opentelemetry_cpp_targets opentelemetry_trace opentelemetry_logs http_client_curl )
   set(opentelemetry_CMAKE_ARGS -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+	                       -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
```

Comment:
nit: there is tab and spaces mixed in the line?

Reply:
Thanks fixed.